server/models/article.js

```diff
@@ -15,17 +15,32 @@ export default (sequelize, DataTypes) => {
     body: {
       allowNull: false,
       type: DataTypes.STRING,
+    },
+    rating: {
```

Comment:
How does this field get updated?

Reply:
We can reference the ID of the article and then update its rating. For instance, 
`UPDATE Articles SET rating = 4 WHERE ID = 10;`

Reply:
Won't it be easier, better and maybe faster to have a beforeInsert method that handles that?

Reply:
I have this working now.

Reply:
@mbilesanmi I have this working now.

Reply:
@mrSerious Working without any issues?

Reply:
Yes.

Reply:
I could have demoed it to you but I already took out the controller.